### DISCUSSION TITLE
how to temporarily change theme for one session without modifying config file?

### DISCUSSION THREAD

**Discussion by sgon00** [NvChad/NvChad#3151] [Q&A] [answered]
how to temporarily change the theme for one session without modifying global config file `~/.config/nvim/lua/chadrc.lua`? When I use `<leader>th` to change a theme for the current session. It will always modify the global config file, but what I expect is to use this theme only for this session only. Thanks.

**Comment by siduck** [comment, marked as answer]
```lua
require("nvconfig").base46.theme="nord"
require("base46").load_all_highlights()
```

this will compile theme too, so it'll be there in next session unless u save config file ( which will re-compile base46 stuff based on chadrc opts )

so on vimleavepre event u can add the load...() func
